### DISCUSSION TITLE
Metrics for Grafana.

### DISCUSSION THREAD

**Discussion by apoorv569** [nextcloud/all-in-one#5219] [Ideas] [answered]
Is it possible to expose metrics for Nextcloud and/or other containers that are part of AIO via Prometheus or InfluxDB etc so I can add it to my Grafana server for visualizing stats about my Nextcloud server?

**Comment by szaimen** [comment]
Hi, you could use this container: https://github.com/xperimental/nextcloud-exporter

**Reply by apoorv569** [reply]
I see. Do I have to run this container along side the AIO containers?

BTW why not make this one be a part of Nextcloud AIO? would be a cool addon to AIO.

**Reply by szaimen** [reply]
> I see. Do I have to run this container along side the AIO containers?

Yes

> BTW why not make this one be a part of Nextcloud AIO? would be a cool addon to AIO.

Good idea! [the community container feature](https://github.com/nextcloud/all-in-one/tree/main/community-containers#community-containers) could be used for that. See [this doc](https://github.com/nextcloud/all-in-one/tree/main/community-containers#how-to-add-containers) how to add a container to the list.

**Reply by apoorv569** [reply]
I'll talk to the nextcloud-exporter dev about including it in AIO.

**Reply by martin-77** [reply]
Please keep us updated!
And to add it to Nextcloud-AIO sounds wonderfull to me!

**Reply by apoorv569** [reply]
I opened an issue on their repo just now, https://github.com/xperimental/nextcloud-exporter/issues/120

**Comment by Grotax** [comment, marked as answer]
There is a PR for this now: https://github.com/nextcloud/all-in-one/pull/6837

**Reply by szaimen** [reply]
This is now released with v11.8.0 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel

**Reply by apoorv569** [reply]
This is great news.
